hw_16/templates/todo.php

```diff
@@ -0,0 +1,36 @@
+<div id="root">
+    <div class="container">
+        <form method="post" class="input-group mb-3">
+            <a href="/hw_16/todos/delete_all.php" class="btn btn-outline-secondary"
+               type="button" id="button-addon1">Delete All</a>
+            <input class="form-control" placeholder="Enter todo.." name="value">
+            <button class="btn btn-outline-secondary" type="submit">ADD</button>
+            <?php if ($_POST) {
+                $value = mysqli_real_escape_string($conn, $_POST['value']);
+                $user_id = $_SESSION['current_user']['id'];
+
+                $sql = "INSERT INTO todos (value, user_id) VALUES ('$value', $user_id)";
+                mysqli_query($conn, $sql);
+                header('Location: index.php');
+                die();
+            } ?>
+        </form>
+        <?php if ($todos): ?>
+        <ul class="list-group list-unstyled d-flex justify-content-between mb-3">
+            <?php foreach ($todos as $todo): ?>
+            <li class="list-group-item d-flex align-items-center ">
+                <input class="form-check-input pr-5" type="checkbox" id="checkboxNoLabel" value="" aria-label="...">
+                <span class="form-control text-center" type="text" aria-label="Disabled input example" disabled="" id="<?= $todo['user_id'] ?>"><?php echo $todo['value'] ?></span>
+                <form method="post" action="/hw_16/todos/delete_one.php" class="d-flex flex-column">
+                    <input type="hidden" name="<?= $todo['value'] ?>" value="<?= $todo['user_id'] ?>">
```

Comment:
не понял, что ты пытался сделать
тебе просто нужно передать id записи, которую ты хочешь удалить
а в обработчике удалить запись по айди
```suggestion
                    <input type="hidden" name="to_id" value="<?= $todo['id'] ?>">
```